### PR TITLE
ID-83

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
Clean up the readme a bit, add some PGP copy

### CODE REVIEW

**Review by xiangzhe0812 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,19 +7,23 @@ Idem is a open source cross platform mobile application based on the Decentrlaiz
 
 Each time an exchange requests an ID verifycation from a user, the provider charges the exchange a fee.  Further, users are required to provide KYC information and have it verified for each and every onboarding, instead of a portable, reusable verification from a trusted provider.  By locally storing user's verified information, we can enhance the customer onboarding experience and reduce costs incurred by Vendors.
```
Comment (styvo51) [inline README.md:8]: verification

File: README.md [hunk 2]
```diff
@@ -7,19 +7,23 @@ Idem is a open source cross platform mobile application based on the Decentrlaiz
...
 
-Idem has two workflows, 
+## The Tech
+Idem uses a verify of crytographic protocols to sign and encrypt your data,  PGP/GPG encryption is used to securely store data on your device.  Futhermore, Idem uses the Ethereum eliptic curve to sign claims which is inline with the DID foundations verifiable claims schema.  Transactions on the Ethereum Testnetwokr "Kovan" will be signed with the ETH account `0xE4ed9ceF6989CFE9da7c1Eec8c2299141dD9e7cC`
```
Comment (styvo51) [inline README.md]:
```suggestion
Idem uses a verify of cryptographic protocols to sign and encrypt your data,  PGP/GPG encryption is used to securely store data on your device.  Furthermore, Idem uses the Ethereum elliptic curve to sign claims which is inline with the DID foundations verifiable claims schema.  Transactions on the Ethereum Test Network "Kovan" will be signed with the ETH account `0xE4ed9ceF6989CFE9da7c1Eec8c2299141dD9e7cC`
```

File: README.md [hunk 3]
```diff
@@ -7,19 +7,23 @@ Idem is a open source cross platform mobile application based on the Decentrlaiz
...
 
-1. Onboarding / Registering new users (user story 1)
-2. Onboarded users who have not completed any verification (user story 2)
+## Intent
+Idem is designed to be used in two ways with third parties such as crypto exchanges. 
 
-Kovan transactions will be signed with the ETH account `0xE4ed9ceF6989CFE9da7c1Eec8c2299141dD9e7cC`
+1. Onboarding / Registering new users who do not have an account on a third party (User Story 1)
```
Comment (styvo51) [inline README.md:16]:
```suggestion
1. Onboarding / Registering new users who do not have an account on a third party platform (User Story 1)
```

Comment (styvo51) [inline README.md]:
```suggestion
1. Onboarding / Registering new users who do not have an account on the third party platform (User Story 1)
```

**Review by ahinchliff — APPROVED** [review]

**Pull request merged**